packages/shared/src/components/auth/PreparingYourFeed.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import LogoIcon from '../../svg/LogoIcon';
+import LogoText from '../../svg/LogoText';
+import classed from '../../lib/classed';
+
+interface PreparingYourFeedProps {
+  isAnimating: boolean;
+}
+
+export const OnboardingContainer = classed(
+  'div',
+  'flex flex-col overflow-x-hidden items-center min-h-[100vh] w-full h-full max-h-[100vh] flex-1 z-max',
+);
+
+export function PreparingYourFeed({
+  isAnimating,
+}: PreparingYourFeedProps): ReactElement {
+  return (
+    <OnboardingContainer className="justify-center">
+      <span className="mb-10 flex flex-row gap-3">
+        <LogoIcon className="w-16" />
+        <LogoText className="w-32" />
+      </span>
+      <span className="flex h-1.5 w-full max-w-[19.125rem] flex-row items-center rounded-12 bg-theme-divider-tertiary px-0.5">
```

Comment:
Do we want to maybe start using the new named colors here already?

Reply:
Yes, it was just not working when I was working with this, so I applied the correct working one.